src/components/colorbar-plot.tsx

```diff
@@ -1,7 +1,52 @@
-import Wireframe from "./wireframe"
+import { getColorForValue } from "../helpers/colormap";
+import { abbrDate } from "../helpers/format-date";
+import { useDataSet } from "../hooks/use-dataset";
+import { useGlobalStateContext } from "../hooks/use-global-state";
 
-function ColorbarPlot() {
-  return <Wireframe>Colorbar Plot</Wireframe>
+import "./colorbar-plot.css";
+
+const ColorbarPlot = () => {
+  const { dataSet } = useDataSet();
+  const {globalState: {selectedMarkers, selectedYMDDate}} = useGlobalStateContext()
+  const { minValue, maxValue, observations } = dataSet;
+
+  const renderTable = () => {
+    return selectedMarkers.map((marker) => {
+      const { position } = marker;
+      return (
+        <table className="colorbar-table">
+          <th colSpan={2}>
+            {marker.position.key}
+          </th>
+          {[...Object.keys(observations)].reverse().map((key) => {
```

Comment:
This works but I wonder if this (and the xyplot) code should change to use the `ymdDates` value of the dataset which is this same set of keys but is more self-documenting.